### ISSUE TITLE
Is it possible to run multiple workers with this wrapper?

### ISSUE THREAD

**Issue by sergshner** [Filsh/yii2-gearman#6]
I want to parallelize some job with multiple worker architecture, but when I'm trying to run second instance of worker with 'php console/yii gearman/start' I get the following error: '
Failed: Process is already running'

**Comment by AnatolyRugalev** [comment]
No, you cannot run more than one workers with this implementation, but I wrote a workaround (code is not public). All you can do is to set environment variable TMPDIR different for each worker. In PHP:

``` php
putenv('TMPDIR=' . Yii::getAlias('@runtime/gearman_worker_1'));
```

Or before running worker in command line:

``` bash
$ export TMPDIR=<path>
```

I am successfully using this workaround for making workers that processing different jobs.

**Comment by comm1x** [comment]
Also need to run multiple workers. Current single worker freezes on one task and do not process other.
